apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx

```diff
@@ -1,31 +1,139 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import PropTypes from 'prop-types';
 import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
 import i18n from '@cdo/locale';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
 import StylizedBaseDialog from '@cdo/apps/componentLibrary/StylizedBaseDialog';
-import CodeReviewGroupsLoader from '@cdo/apps/templates/codeReviewGroups/CodeReviewGroupsLoader';
 import CodeReviewGroupsStatusToggle from '../codeReviewGroups/CodeReviewGroupsStatusToggle';
+import CodeReviewGroupsManager from '@cdo/apps/templates/codeReviewGroups/CodeReviewGroupsManager';
 
 const DIALOG_WIDTH = 1000;
+const STATUS_MESSAGE_TIME_MS = 5000;
+
+const SUBMIT_STATES = {
+  DEFAULT: 'default',
+  SUBMITTING: 'submitting',
+  SUCCESS: 'success',
+  ERROR: 'error'
+};
+
+const LOADING_STATES = {
+  LOADING: 'loading',
+  LOADED: 'loaded',
+  ERROR: 'error'
+};
 
 export default function ManageCodeReviewGroups({
   buttonContainerStyle,
-  sectionId
+  dataApi
 }) {
+  const [groups, setGroups] = useState([]);
+  const [groupsHaveChanged, setGroupsHaveChanged] = useState(false);
+  const [submitStatus, setSubmitStatus] = useState(SUBMIT_STATES.DEFAULT);
+  const [loadingStatus, setLoadingStatus] = useState(LOADING_STATES.LOADING);
   const [isDialogOpen, setIsDialogOpen] = useState(false);
 
   const openDialog = () => setIsDialogOpen(true);
   const onDialogClose = () => setIsDialogOpen(false);
 
+  const setInitialGroups = groups => setGroups(groups);
```

Comment:
Why do we need this wrapper? Could we just call `setGroups` directly?

Reply:
I think this is a holdover from when I was passing this function down to `CodeReviewGroupsLoader`. Will update.